sched/worker/noop.go

```diff
@@ -0,0 +1,27 @@
+package worker
+
+import (
+	"github.com/scootdev/scoot/cloud/cluster"
+	"github.com/scootdev/scoot/sched"
+)
+
+type noopWorkerController struct{}
+
+func (c *noopWorkerController) RunAndWait(task sched.TaskDefinition) error {
+	return nil
```

Comment:
LocalNode used to have a random sleep in here, to make the demo more realistic.  Can you please add this back